src/components/util/RivetIcons.tsx

```diff
@@ -0,0 +1,88 @@
+import * as React from 'react'
+
+export const svg16 = (className: string = "", node: React.ReactNode) =>
```

Comment:
This name is kind of confusing (you have to look into the code of the rendered component to see that the `16` in the name refers to the size). Can it just be named `svgBase` or something similar?

Reply:
We may also want to consider making this size customizable (maybe not in this PR since there isn't an outstanding use case for it), but it's another reason not to include the size in the constant name.

Reply:
Changing to SFC with DefaultProps, which should make the sizing adjustable via props.